tests/testutils/src/main/kotlin/io/element/android/tests/testutils/fake/FakeTemporaryUriDeleter.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2024 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ * Please see LICENSE in the repository root for full details.
+ */
+
+package io.element.android.tests.testutils.fake
+
+import android.net.Uri
+import io.element.android.libraries.androidutils.file.TemporaryUriDeleter
+import io.element.android.tests.testutils.lambda.lambdaError
+
+class FakeTemporaryUriDeleter(
+    val deleteCallback: (uri: Uri?) -> Unit = { lambdaError() }
```

Comment:
name it `deleteLambda` like we do elsewhere?

Reply:
b13605de7aa9a24f18bdb66bfbccb93417ad2794